pkgs/applications/editors/vim/plugins/build-vim-plugin.nix

```diff
@@ -27,8 +26,7 @@ rec {
     }@attrs:
     let
       drv = stdenv.mkDerivation (attrs // {
-        name = namePrefix + name;
```

Comment:
question: Is `buildVimPlugin` considered stable?
I highly doubt anyone has been using the `namePrefix`, but maybe it's a good idea to default it to an empty string, with a deprecation warning if set, just in case?

Reply:
you are the voice of reason, I should not break stuff this close to a release. I will need to remove some of these deprecation notices after the release since the vim ecosystem has several.
Is it better like this ?

Reply:
Looks good to me.